src/styles/hub/admin-bar.scss

```diff
@@ -18,18 +18,17 @@
 	}
 
 	/* the bell icon */
-	.ab-item {
+	.hub-icon {
 		appearance: none;
 		height: 32px;
 		display: block;
-		padding: 0 10px;
```

Comment:
The `ab-empty-item` class provides the padding. The `ab-empty-item` class is applied to menu items that do not have a link.

https://github.com/WordPress/wordpress-develop/blob/6d4fc50edc0f0a57b53577d421dcaca5d943a227/src/wp-includes/class-wp-admin-bar.php#L563-L569